ee/fe/modules/500-openvpn/hooks/generate_password.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2021 Flant JSC
+Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+*/
+
+package hooks
+
+import (
+	"encoding/json"
+
+	"github.com/flant/addon-operator/pkg/module_manager/go_hook"
+	"github.com/flant/addon-operator/sdk"
+
+	"github.com/deckhouse/deckhouse/go_lib/pwgen"
+)
+
+var _ = sdk.RegisterFunc(&go_hook.HookConfig{
+	Queue:        "/modules/openvpn/generate_password",
+	OnBeforeHelm: &go_hook.OrderedConfig{Order: 10},
+}, generatePassword)
+
+func generatePassword(input *go_hook.HookInput) error {
```

Comment:
Let's also move this hook to the shared library (and replace it for other modules).